config/meteor.yaml.sample

```
@@ -0,0 +1,7 @@
+LOG_LEVEL: hehehehehe
+STATSD_ENABLED: false
+STATSD_HOST: "localhost:8005"
```

```suggestion
STATSD_HOST: "localhost:8125"
```

8125 is statsd default port